doc/api/process.md

```diff
@@ -1896,6 +1896,102 @@ a code.
 Specifying a code to [`process.exit(code)`][`process.exit()`] will override any
 previous setting of `process.exitCode`.
 
+## `process.features.cached_builtins`
+
+<!-- YAML
+added: REPLACEME
```

Comment:
https://github.com/nodejs/node/pull/27311
```suggestion
added: v12.0.0
```

Reply:
![lol](https://github.com/user-attachments/assets/b6a9f805-a0aa-4263-82ae-c231ac9e794c)

Reply:
I see I have reverted the commit with the suggestions with a rebase

Reply:
I've opened https://github.com/nodejs/node/pull/54982 to fix this.